### PR TITLE
Update system-requirements.md

### PR DESCRIPTION
**Pull request by TinaUll**
Hi @osisoft-jcoffman, @ewillis-osi, and @cmorgan-osi, using this branch for the final RDBMS review. Please make any edits in this branch. Thank you!

### CODE REVIEW

**Review by ewillis-osi — APPROVED** [review]

File: content/installation/system-requirements.md [hunk 1]
```diff
@@ -10,8 +10,8 @@ PI Adapter for RDBMS is supported on a variety of platforms and processors. Inst
 |-------------------|-------------|----------------------------------|-------------|
 | Windows 10 Enterprise <br>Windows 10 IoT Enterprise | x64 | `RDBMS_win10-x64.msi`     | Intel/AMD 64-bit processors |
 | Debian 9, 10 <br>Ubuntu 18.04, 20.04 | x64 | `RDBMS_linux-x64.deb`     | Intel/AMD 64-bit processors |
-| Debian 9, 10 <br>Ubuntu 18.04, 20.04 | ARM32 | `RDBMS_linux-arm.deb`  | ARM 32-bit processors |
-| Debian 9, 10 <br>Ubuntu 18.04, 20.04 | ARM64 | `RDBMS_linux-arm64.deb`  | ARM 64-bit processors |
```
Comment (cmorgan-osi) [inline content/installation/system-requirements.md:14]: I'm not sure Ubuntu 18.04 is to be removed from ARM64. Chris suggested here that we remove it from ARM32:
https://teams.microsoft.com/l/message/19:fc741e2ed69347b38647c678db8dc4e1@thread.skype/1632502363613?tenantId=a445ee81-2b91-4806-883b-1dc673d59147&groupId=25d0fd61-3800-49e5-9ad3-2fd7ca1fb67c&parentMessageId=1632502363613&teamName=Integration%20Technologies&channelName=AdapterFrameworkSupport&createdTime=1632502363613

**Comment by cmorgan-osi** [comment]
There may be parallel conversations going on about the supported OS's in the system requirements. If we are sure we're removing ARM64 Ubuntu 18.04, then it looks good to me!

**Comment by cmorgan-osi** [comment]
Sorry, I did not mean to close.

**Review by cmorgan-osi — APPROVED** [review]

**Review by osisoft-jcoffman — APPROVED** [review]

**Comment by TinaUll** [comment]
@osisoft-mbishop This one is ready for Doc QA. Please make any edits/comments in this branch. Thank you!

**Pull request merged**